include/openssl/ec.h

```diff
@@ -1568,6 +1568,7 @@ int ERR_load_EC_strings(void);
 # define EC_R_SLOT_FULL                                   108
 # define EC_R_UNDEFINED_GENERATOR                         113
 # define EC_R_UNDEFINED_ORDER                             128
+# define EC_R_UNKNOWN_COFACTOR                            136
```

Comment:
This should be 164 to maintain ABI compatibility between 1.1.0 and 1.1.1

Reply:
Fixed by 9b29942